fedimint-core/src/api.rs

```diff
@@ -421,6 +468,16 @@ impl AsRef<dyn IGlobalFederationApi + 'static> for DynGlobalApi {
 }
 
 impl DynGlobalApi {
+    pub fn from_pre_peer_id_endpoint(url: SafeUrl) -> Self {
+        // PeerIds are used only for informational purposes, but just in case, make a
+        // big number so it stands out
+        GlobalFederationApiWithCache::new(WsFederationApi::new(vec![(PeerId::from(1024), url)]))
+            .into()
+    }
+
+    pub fn from_single_endpoint(peer: PeerId, url: SafeUrl) -> Self {
+        GlobalFederationApiWithCache::new(WsFederationApi::new(vec![(peer, url)])).into()
+    }
```

Comment:
Is it really worth having two of these functions?

Reply:
Doesn't matter too much, just wanted a simplest patch possible. We can clean up with time.